fvtest/tril/tril/ilgen.cpp

```diff
@@ -184,11 +185,34 @@ TR::Node* Tril::TRLangBuilder::toTRNode(const ASTNode* const tree) {
         }
      }
      else if (opcode.isLoadIndirect() || opcode.isStoreIndirect()) {
-         auto offset = tree->getArgByName("offset")->getValue()->get<int32_t>();
+         // If not specified, offset will default to zero. 
+         int32_t offset = 0; 
+         if (tree->getArgByName("offset")) {
+            offset = tree->getArgByName("offset")->getValue()->get<int32_t>();
+         } else { 
+            // warning("Should specify offset on indirect load and stores -- has defaulted to zero"); 
```

Comment:
Maybe put a trace message here? Just to signal that the offset has defaulted to 0.

Reply:
Done -- I've reworked the tracing in that area as part of 751134f; 

output now looks like 

    Creating vstorei from ASTNode 0x7fbd59402c00
      is indirect store/load with offset 0  of vector type VectorDouble